docs/Makefile

```diff
@@ -23,14 +24,19 @@ build: build-examples build-helm
 	cp -r ./swagger-ui site/html/rest-api
 	cp ../proto/build/swagger/determined/api/v1/api.swagger.json  site/html/rest-api/
 
+.PHONY: set-static-swagger
+set-static-swagger:
+	rm $(swagger_main_script)
+	cp ./swagger-ui/swagger-ui-main-deploy.js $s(swagger_main_script)
+
 .PHONY: clean
 clean:
 	rm -rf site
 
 .PHONY: publish
 publish: clean
 	$(MAKE) build
-	cp ./swagger-ui/swagger-ui-main-deploy.js ./site/html/rest-api/swagger-ui-main.js
+	$(MAKE) set-static-swagger
```

Comment:
because we have two separate build artifacts (at least for swagger) we want a build A bundled with det cluster but we want to modify that when we publish to a static server.

Reply:
ah, now I understand what `set-static-swagger` means.

Reply:
This seems sorta reasonable but a nicer solution would be to just generate two separate build artifacts, rather than modify one of them at `publish` time.

Reply:
hmm we're swapping a config file for the build we publish I thought it would be simpler to keep it at one. but two works two you're thinking build two at build time?